### PR TITLE
Zwave Database files for Evolve LCD1

### PR DESCRIPTION
**Pull request by paixaop**
Chris

Here are the database updates for the Evolve LCD1 5 button panel

Thanks!

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Scene Control Toggle</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">BASICSET Toggle</Label>
+			</Item>
+			<Item>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml]: Just checking that there shouldn't be an option 3? In param 2 there is and it looks basically the same as param 1...

File: bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+				<Label lang="en">Scene Control Momentary</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Scene Control Toggle</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">BASICSET Toggle</Label>
+			</Item>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml]: Again - there's no param 3 - is this correct?

File: bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml [hunk 3]
```diff
@@ -0,0 +1,647 @@
...
+				<Value>18</Value>
+				<Label lang="en">CES3DEMO</Label>
+			</Item>
+		</Parameter>
+		<Parameter>
+			<Index>20</Index>
+			<Type>short</Type>
+			<Size>1</Size>
+			<Default>15</Default>
+			<Label lang="en">Display Timeout</Label>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml]: There probably should be Minimum and Maximum specified in here?

**Comment by cdjackson** [comment]
I note that this device has lots of association groups that you've all tagged with ```<SetToController>```. Do they really all need to be tagged this way? (fine if they are, but if not it might cause some problems depending on what the associations are sending).

File: bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml (hunk 1 above)
Comment (paixaop) [inline bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml]: Chris params 1 and 5 do not have option 3 (N/A) in manual

Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml]: > Chris params 1 and 5 do not have option 3 (N/A) in manual
> 
> 

Ok - thanks. Just checking :)

**Comment by paixaop** [comment]
Manufacturer does not specify min/max for paramter 20.
Not sure about the SetToController, they are not clear about it in the manual.

**Comment by paixaop** [comment]
After some testing I think we should leave the SetToController in the association groups as the panel will not inform it of any button presses otherwise.

**Comment by cdjackson** [comment]
Ok - that’s fine… Thanks.

**Comment by cdjackson** [comment]
I can't merge this as there is a merge conflict. Can you please resolve the conflict?

**Comment by paixaop** [comment]
Chris I did a merge with all the latest changes in openhab/master i hope this cleans it up so you can merge it

**Comment by cdjackson** [comment]
This no looks in a bad way - there's a lot of changes in here that aren't associated with the zwave binding so we need to resolve this.

I think there are still a few changes to make as well, but it's hard to tell as the changeset is now so large. IF you can consolidate this into a single PR that would be good (after sorting out the mess with all the files).

Thanks
Chris

**Pull request closed without merging**